core-bundle/contao/drivers/DC_Table.php

```diff
@@ -5185,15 +5192,13 @@ protected function sortMenu()
 			return strnatcmp($a->ascii()->toString(), $b->ascii()->toString());
 		});
 
-		return '
-<div class="tl_sorting tl_subpanel">
-<strong>' . $GLOBALS['TL_LANG']['MSC']['sortBy'] . ':</strong>
-<div class="tl_select_wrapper"" data-controller="contao--choices">
-<select name="tl_sort" id="tl_sort" class="tl_select">
-' . implode("\n", $options_sorter) . '
-</select>
-</div>
-</div>';
+		$parameters = array(
+			'options' => $options_sorter
```

Comment:
The keys are not used in the template. Should we use a list instead?
```suggestion
			'options' => array_values($options_sorter),
```